draft-tjhai-ipsecme-hybrid-qske-ikev2-04.xml

```diff
@@ -0,0 +1,994 @@
+            the performed key exchanges are classical (EC)DH ones, but for some reasons
+            (e.g. policy requirements) it is essential to perform multiple of them.
+            </t>
+		</section>
+
+    	<section title="Changes" >
+            <t>RFC EDITOR PLEASE DELETE THIS SECTION.</t>
+
+            <t> Changes in this draft in each version iterations.</t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-04</t>
+            <t><list style="symbols">
+            <t>Clarification about key derivation in case of multiple key exchanges in CREATE_CHILD_SA is added.</t>
+            <t>Resolving rekey collisions in case of multiple key exchanges is clarified.</t>
+            </list></t>
+        
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-03</t>
+            <t><list style="symbols">
+            <t>Using multiple key exchanges CREATE_CHILD_SA is defined.</t>
+            </list></t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-02</t>
+            <t><list style="symbols">
+        	<t>Use new transform types to negotiate additional key exchanges,
+        	rather than using the KE payloads of IKE SA.</t>
+         	</list></t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-01</t>
+            <t><list style="symbols">
+     		<t>Use IKE_INTERMEDIATE to perform multiple key exchanges in succession.</t>
+     		<t>Handle fragmentation by keeping the first key exchange (a standard
+     		IKE_SA_INIT with a few extra notifies) small, and encrypting the rest
+     		of the key exchanges.</t>
+    
+     		<t>Simplify the negotiation of the ‘extra’ key exchanges.</t>
+     	    </list></t>
+    
+            <t>draft-tjhai-ipsecme-hybrid-qske-ikev2-00</t>
+            <t><list style="symbols">
+    		<t>We added a feature to allow more than one post-quantum key
+            exchange algorithms to be negotiated and used to exchange a post-
+            quantum shared secret.</t>
+    		<t>Instead of relying on TCP encapsulation to deal with IP level
+    		fragmentation, we introduced a new key exchange payload that can
+    		be sent as multiple fragments within IKE_SA_INIT message.</t>
+    	    </list>
+            </t>
+        </section>
+    
+    	<section title="Document Organization" >
+            <t>
+            The remainder of this document is organized as follows.  <xref target="design"/>
+            summarizes design criteria.  <xref target="framework"/> describes how
+    	    post-quantum key exchange is performed between two IKE peers and how
+    	    keying materials are derived for both SAs and child SAs.  A summary of alternative
+    	    approaches that have been considered, but later discarded, are described
+    	    in <xref target="altdesign"/>.  <xref target="IANA"/>
+            discusses IANA considerations for the namespaces introduced in this
+    	    document, and lastly <xref target="security"/> discusses security considerations.
```

Comment:
Since we have moved the alternative design to Appendix, perhaps we should re-arrange the sentence slightly:

<xref target="IANA"/> discusses IANA considerations for the namespaces introduced in this
document, and <xref target="security"/> discusses security considerations.  Lastly, a summary
of alternative approaches that have been considered, but later discarded, are described in
<xref target="altdesign"/>.

Reply:
OK, thank you for catching this up.

Reply:
Do you want me to change the file or you'll change it by yourself?

Reply:
I'm happy to do it :-)